src/transformers/generation_utils.py

```diff
@@ -649,7 +722,7 @@ def _get_stopping_criteria(self, max_length: Optional[int], max_time: Optional[f
     @torch.no_grad()
     def generate(
         self,
-        input_ids: Optional[torch.LongTensor] = None,
+        inputs: Optional[torch.Tensor] = None,
```

Comment:
Full backward compatibility is kept here

Reply:
Should we document somewhere that the `inputs` can be replaced by `input_ids`/`pixel_values` etc?

Reply:
I agree mentionning this in the docs would enable Ctrl+F in the docs for the old parameters.

Reply:
Sounds good!